include/SDL_video.h

```diff
@@ -237,6 +237,7 @@ typedef enum
     SDL_GL_STEREO,
     SDL_GL_MULTISAMPLEBUFFERS,
     SDL_GL_MULTISAMPLESAMPLES,
+    SDL_GL_FLOATBUFFERS,
```

Comment:
You are changing the abi by inserting here instead of appending: Wrongest thing to do.

Reply:
Ah shoot sorry, I wasn't thinking about the ABI ramifications. I'll reshuffle this.

Reply:
I've moved the new entry to the end of the enum. I assume the changes to ```SDL_sysvideo.h``` are ok, as that is an internal struct?